### ISSUE TITLE
iOS: "Could not find the service definitions manifest. No custom services will be available."

### ISSUE THREAD

**Issue by eseidelGoogle** [flutter/flutter#4873]
```
 %flutter run --debug                                                                                                                      [/src/game]
Running lib/main.dart on flutter_ipod_touch...
vm-service: Isolate creation error: (null)
[INFO:FlutterDynamicServiceLoader.mm(129)] Could not find the service definitions manifest. No custom services will be available.
[INFO:FlutterDynamicServiceLoader.mm(107)] 0 custom service definitions registered
vm-service: Isolate creation error: (null)
[INFO:FlutterDynamicServiceLoader.mm(129)] Could not find the service definitions manifest. No custom services will be available.
[INFO:FlutterDynamicServiceLoader.mm(107)] 0 custom service definitions registered
vm-service: Isolate creation error: (null)
[INFO:FlutterDynamicServiceLoader.mm(129)] Could not find the service definitions manifest. No custom services will be available.
[INFO:FlutterDynamicServiceLoader.mm(107)] 0 custom service definitions registered
[INFO:FlutterDynamicServiceLoader.mm(129)] Could not find the service definitions manifest. No custom services will be available.
[INFO:FlutterDynamicServiceLoader.mm(107)] 0 custom service definitions registered
[INFO:FlutterDynamicServiceLoader.mm(129)] Could not find the service definitions manifest. No custom services will be available.
[INFO:FlutterDynamicServiceLoader.mm(107)] 0 custom service definitions registered
```

**Comment by abarth** [comment]
Yeah, we should probably remove it.  It's the old way of integrating services.

**Comment by abarth** [comment]
Fixed by 07b59dff2c1906a0c637a6088697bb449f6074b7
